client/modules/module.js

```diff
@@ -90,16 +85,10 @@ define(function(require) {
           klass = clientSide;
         },
         require: fakeRequire.createEnvironment(exposedNodeModules),
-        ServerModuleInterface: moduleInterface.Server,
-        ClientModuleInterface: moduleInterface.Client,
-        Object: Object,
         Surface: Surface,
         CanvasSurface: CanvasSurface,
         P5Surface: P5Surface,
         ThreeJsSurface: ThreeJsSurface,
-        geometry: geometry,
```

Comment:
Now that geometry is not a global can you also remove it from the jshint whitelist?